plugin.py

```diff
@@ -16,3 +16,11 @@ class LspTailwindcssPlugin(NpmClientHandler):
     server_binary_path = os.path.join(
         server_directory, 'extension', 'dist', 'server', 'index.js'
     )
+
+    @classmethod
+    def on_client_configuration_ready(cls, configuration: dict) -> None:
+        configuration["initializationOptions"].setdefault("userLanguages", {
+            # the server requires the existance of "userLanguages" in initializationOptions
+            # userLanguages is a VS Code specific setting
+        })
```

Comment:
Are you sure that we have to set this up? The code seems to have proper checks that should make it work even if not passed: https://github.com/tailwindlabs/tailwindcss-intellisense/blob/8fce24db9c480a9615e175fb3acaac91f7a0a7c1/packages/tailwindcss-intellisense/src/lsp/server.ts#L93-L97

Reply:
I saw that code and saw that it has fallbacks.
If i dont send userLanguages, the server does initialaze successfully. But if i trigger the AC the server request fails. 

If you want you can checkout this PR locally, and comment out this code I added, and see if you can have completions returned(if you dont want to bother, I will try it later today again)

Reply:
I guess I can't be bothered to create a proper project. It looks like it needs some configuration file.

Also noticed the issues with completion triggers. See another comment. That sounds like relevant to what you are describing but not really related to this issue.

Reply:
@rchl they changed the code. there is [no fallback now](https://github.com/tailwindlabs/tailwindcss-intellisense/blob/master/src/server.ts#L159) :) 
so I guess we need to seed the userLanguages in the init options

Reply:
Sounds like something that should be reported and fixed. :)